prow/plugins/label/label.go

```diff
@@ -96,14 +121,15 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 
 // getLabelsFromGenericMatches returns label matches with extra labels if those
 // have been configured in the plugin config.
-func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string) []string {
+func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string, invalidLabels *[]string) []string {
 	if len(additionalLabels) == 0 {
 		return nil
 	}
 	var labels []string
 	for _, match := range matches {
 		parts := strings.Split(match[0], " ")
 		if ((parts[0] != "/label") && (parts[0] != "/remove-label")) || len(parts) != 2 {
+			*invalidLabels = append(*invalidLabels, match[0])
```

Comment:
`match[0]` is the entire command, not a label value (e.g. `/label foo` instead of `foo`).
I think the only time we hit this case is when `/label` or `/remove-label` is used without an argument or with more than one argument.

Reply:
As you pointed out, that case is the reason I picked `match[0]`. I think this makes feedback specific for the end-user.